### PR TITLE
update Indonesian translation

### PR DESCRIPTION
**Pull request by HongjunTew**

### CODE REVIEW

**Comment by lbk2907** [comment]
I will continue on tomorrow (GMT+0800) to remove the unneeded translations.

**Comment by lbk2907** [comment]
I'll continue to remove the unneeded translations. I'll leave 1 more comment here after my last commit later. When you receive that comment, it's mean it are ready to merge to master.

File: Client/TeamTalkAndroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <resources>
 
     <string name="app_name">TeamTalk</string>
-    <string name="title_activity_server_list">TeamTalk 5 Servers</string>
+    <string name="title_activity_server_list">TeamTalk</string>
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/res/values/strings.xml:5]: What is the effect of this change?

**Comment by lbk2907** [comment]
I just change the name from TeamTalk 5 Servers to TeamTalk.

**Comment by lbk2907** [comment]
If needed, I can revert back.

**Comment by bear101** [comment]
But why did you make this change?

**Comment by lbk2907** [comment]
I try to explain cause of my English. A few people has ask me. After they open the app, when they swipe to the left from the new server button, screen reader will read "TeamTalk 5 Servers". Have any way to change it to match the app name that is "TeamTalk"? And I said I try to make a pull request for it in the same time I’m updating the translation.

**Comment by lbk2907** [comment]
Here you go. I'll push the last commit in this contribution. It ready to be merge.

**Comment by CoBC** [comment]
I'm not sure what to think about the renaming of the server list screen on the Android client. I've never heard anyone complain about it, and if you really want to change it, I think you need to use TeamTalk5 instead of just TeamTalk.
@bear101 what do you think?

**Comment by lbk2907** [comment]
> I'm not sure what to think about the renaming of the server list screen on the Android client. I've never heard anyone complain about it, and if you really want to change it, I think you need to use TeamTalk5 instead of just TeamTalk. @bear101 what do you think?

It will be good in terms of speech making things smoother for screen reader
@CoBC

**Pull request merged**